tools/loader/ImageClassifier.cpp

```diff
@@ -101,6 +112,25 @@ static bool getNextImageFilenames(std::vector<std::string> *filenames) {
   return !filenames->empty();
 }
 
+/// Generate in \p imageList the list of filenames corresponding to the next
+/// mini-batch of size \p miniBatchSize extracted from \p totalImageList at
+/// index \p minibatchIndex. /returns true if the index is valid, false
+/// otherwise.
```

Comment:
nit: Please update to also mention `minibatchIndex` is incremented by `miniBatchSize`, as that is not clear from the caller's perspective.

Reply:
Done.